### PR TITLE
Adds more atmos logging.

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

Adds more atmos protection, by adding the "protected device" alert to select pipes.
Adds more atmos logging by actually logging those messages in the admin logs.

### CODE REVIEW

**Comment by simonset55** [comment]
GOOD SHIT

File: code/modules/atmospherics/machinery/pipes/pipes.dm [hunk 1]
```diff
@@ -74,6 +75,9 @@
 	parent = P
 
 /obj/machinery/atmospherics/pipe/Destroy()
+	if(adminlog == 1)
+		message_admins("[key_name_admin(usr)]<A HREF='?_src_=holder;adminmoreinfo=\ref[usr]'>?</A> (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[usr]'>FLW</A>) DESTROYED A VERY IMPORTANT PIPE. <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[loc.x];Y=[loc.y];Z=[loc.z]'>(JMP)</a> This alert will not show again for this device.") //[usr] DESTROYED A PROTECTED PIPE YOU FUCKING MORONS ~~JohnCena1469
```
Comment (Jc1469) [inline code/modules/atmospherics/machinery/pipes/pipes.dm:79]: >  This alert will not show again for this device.

> /obj/machinery/atmospherics/pipe/Destroy()

you don't say

Comment (Amelia0010) [reply code/modules/atmospherics/machinery/pipes/pipes.dm:79]: It's also in the one for pumps, so if I remove it now people'll think this one triggers again

Comment (Jc1469) [reply code/modules/atmospherics/machinery/pipes/pipes.dm:79]: I imagine it's in the pumps because they trigger when turned off and on.

Comment (nichlas0010) [reply code/modules/atmospherics/machinery/pipes/pipes.dm]: `/obj/machinery/atmospherics/components/binary/pump/Destroy()
	if(adminlog == 1)
		message_admins("[key_name_admin(usr)]<A HREF='?_src_=holder;adminmoreinfo=\ref[usr]'>?</A> (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[usr]'>FLW</A>) destroyed protected device '[src]'. <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[loc.x];Y=[loc.y];Z=[loc.z]'>(JMP)</a> This alert will not show again for this device.")
		log_admin("[usr] destroyed protected device `[src]`")`

Comment (Amelia0010) [reply code/modules/atmospherics/machinery/pipes/pipes.dm:79]: This alert will not show again for this device

/obj/machinery/atmospherics/components/binary/pump/Destroy()

Comment (Jc1469) [reply code/modules/atmospherics/machinery/pipes/pipes.dm:79]: :thinking: 

whatever. pr is fine either way.   im sure cruix will find something wrong somewhere though.


Also, might aswell put that screenshot in here.

**Pull request merged**